packages/apps/src/contexts/activity.ts

```diff
@@ -285,4 +291,12 @@ export class ActivityContext<T extends Activity = Activity> implements IActivity
       signout: this.signout.bind(this),
     };
   }
+
+  private stripOutThreadId(conversationId: string): string {
```

Comment:
very minor thing, but this could be a useful util to but on the `Activity` class in the `api` package if we think devs may need to do it ever. If not disregard.